core/electron/src/backend/ElectronHost.ts

```diff
@@ -259,7 +259,7 @@ export class ElectronHost {
       this._ipc = new ElectronIpc();
       const app = this.app;
       if (!app.isReady())
-        this.electron.protocol.registerSchemesAsPrivileged([{ scheme: "electron", privileges: { standard: true, secure: true } }]);
+        this.electron.protocol.registerSchemesAsPrivileged([{ scheme: "electron", privileges: { standard: true, secure: true, supportFetchAPI: true } }]);

```

Comment:
Nitpicky, but would be nice to see the array of one object formatted on multiple lines.

Reply:
Agreed - done.